# 2d image and point cloud messages as input data to pointing 
sensor_msgs/Image img_msg
sensor_msgs/PointCloud2 depth_points
---
# Output data from the openpose server
float32[] head
float32[] right_shoulder
float32[] left_shoulder
float32[] left_elbow
float32[] right_elbow
float32[] spine_chest
float32[] left_hand_tip
float32[] right_hand_tip
float32[] left_wrist
float32[] right_wrist
sensor_msgs/Image open_pose_output_image_msg